test/WebJobs.Script.Tests.Shared/TestHelpers.cs

```diff
@@ -141,7 +142,18 @@ public static void ClearHostLogs()
             if (lastLogFile != null)
             {
                 string[] logs = await ReadAllLinesSafeAsync(lastLogFile.FullName);
-                return new Collection<string>(logs.ToList());
+                var rgx = new Regex(@"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}\s");
```

Comment:
This test helper should just be returning the unadulterated log lines. I don't think we should munge them like this. Let's just update the test, ensuring that we're preserving test intent.

Reply:
Yeah, I was going between these two options. I went with this option because the other one seems like it just sweeps the underlying issues with testing our logs under the rug.

The issue is that we don't really have a structured log format, beyond the timestamp. I was attempting to fix the helper, which right now returns lines from a file, not logs. I could just update the test to point to a different index, or I could format the object before we print it so it doesn't have new lines, but the test helper would still cause this issue with someone in the future.

Using timestamp as the only signal for a new log is really flimsy, to be fair. I can bodge the test so it passes, but I do think it's weird to make an assumption that there is no new lines in the log format.